### PR TITLE
Staging to Merge Staging Production

### PR DESCRIPTION
**Pull request by prateek-giddh**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/app.routes.ts [hunk 1]
```diff
@@ -97,7 +97,7 @@ export const ROUTES: Routes = [
             { path: 'permissions', loadChildren: './permissions/permission.module#PermissionModule', canActivate: [NeedsAuthorization] },
             { path: 'settings', loadChildren: './settings/settings.module#SettingsModule', canActivate: [NeedsAuthorization] },
             { path: 'manufacturing', loadChildren: './manufacturing/manufacturing.module#ManufacturingModule', canActivate: [NeedsAuthorization] },
-            { path: 'accounting-voucher', loadChildren: './accounting/accounting.module#AccountingModule' },
+            // { path: 'accounting-voucher', loadChildren: './accounting/accounting.module#AccountingModule' },
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/app.routes.ts:100]: will this commented code reuse in future?

File: apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ export const Gstr1TransactionType = [
     { label: 'Credit Notes', value: 'credit-notes' },
     { label: 'Debit Notes', value: 'debit-notes' },
     { label: 'Advance Receipt', value: 'advance-receipt' },
-    { label: 'Adjusted advance receipt', value: 'adjusted-advance-receipt' },
+    // { label: 'Adjusted advance receipt', value: 'adjusted-advance-receipt' },
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.ts:20]: will this commented code reuse in future?

File: apps/web-giddh/src/app/models/defaultMenus.ts [hunk 3]
```diff
@@ -117,9 +117,9 @@ export let DEFAULT_MENUS_ORIGINAL: IUlist[] = [
     {
         type: 'MENU', name: 'Invoice', uniqueName: '/pages/invoice/preview/sales', isRemoved: false, pIndex: 2, additional: { tab: 'sales', tabIndex: 0 }
     },
-    {
-        type: 'MENU', name: 'Journal Voucher', uniqueName: '/pages/accounting-voucher', isRemoved: false, pIndex: 1
-    },
+    // {
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/models/defaultMenus.ts:120]: will this commented code reuse in future?

File: apps/web-giddh/src/app/shared/header/header.component.html [hunk 4]
```diff
@@ -365,8 +365,8 @@ <h6 class="mrB1">MENU
                                         </li>
                                         <li><a (click)="analyzeOtherMenus('contact/vendor', {tab: 'vendor', tabIndex: 0})">Vendor</a>
                                         </li>
-                                        <li><a (click)="analyzeOtherMenus('accounting-voucher')">Journal Voucher</a>
-                                        </li>
+                                        <!-- <li><a (click)="analyzeOtherMenus('accounting-voucher')">Journal Voucher</a>
+                                        </li> -->
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/shared/header/header.component.html:369]: will this commented code reuse in future?

File: apps/web-giddh/src/app/app.routes.ts (hunk 1 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/app.routes.ts:100]: It's JV code commented for some time as new JV module is under development.

File: apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.ts (hunk 2 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/gst/filing/tabs/overview/view-transactions/view-transactions.component.ts:20]: It's JV code commented for some time as new JV module is under development.

File: apps/web-giddh/src/app/models/defaultMenus.ts (hunk 3 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/models/defaultMenus.ts:120]: It's JV code commented for some time as new JV module is under development.

File: apps/web-giddh/src/app/shared/header/header.component.html (hunk 4 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/shared/header/header.component.html:369]: It's JV code commented for some time as new JV module is under development.

**Pull request merged**